.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -25,70 +25,70 @@ body:
       - cmd/tracegen
       - confmap/provider/s3provider
       - examples/demo
-      - exporter/alibabacloudlogservice
```

Comment:
Thank you. I ran the make target, because previously I had a related a failure and that's how CI said to handle it. But I just pushed with these files reverted to main.

Reply:
When I use ./ISSUE_TEMPLATE/ from main, the CI says to run `make generate-gh-issue-templates`. When I do it makes a 1k+ line diff. (see https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/3623577136/jobs/6109740532) The main difference I see is that the one I generated locally includes the suffix of the component type whereas the one on main doesn't include that. I'm sure something is out of date locally. I'm now pushing with one line changes by hand to see if I can appease this CI check.